lib/list/getCSVData.js

```diff
@@ -0,0 +1,110 @@
+var _ = require('lodash');
+var listToArray = require('list-to-array');
+
+/**
+ * Applies option field transforms to get the CSV value for a field
+ */
+
+function transformFieldValue (field, item, options) {
```

Comment:
Can we unit test this function? Seems nicely encapsulated, would make it _a lot_ clearer which inputs it can take.